shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -28,7 +28,7 @@
     <ul class="menubar">
       {{#each items}}
         {{#unless onlyPopup}}
-          <li class="{{name}}">
+          <li {{inlineStyle}} class="{{name}}">
```

Comment:
I'd rather see this read `style="{{inlineStyle}}"` and have `inlineStyle` return a string